### ISSUE TITLE
Critical React Warnings Regarding `myNav`

### ISSUE THREAD

**Issue by mgamlem3** [mgamlem3/Energy-Dashboard#85]
**Describe the bug**
Copy from browser console
```
Warning: <myNav /> is using incorrect casing. Use PascalCase for React components, or lowercase for HTML elements.
    in myNav (created by Nav)
    in Nav (created by Header)
    in div (created by Header)
    in Header (created by App)
    in div (created by App)
    in App
```
and
```
Warning: The tag <myNav> is unrecognized in this browser. If you meant to render a React component, start its name with an uppercase letter.
    in myNav (created by Nav)
    in Nav (created by Header)
    in div (created by Header)
    in Header (created by App)
    in div (created by App)
    in App
```

**Expected behavior**
These critical warnings should not be present. Follow React pascal case rules.

**To Reproduce**
- Build most recent `testing` version
- Navigate to webpage
- open browser console

**Screenshots**
NA

**Desktop (please complete the following information):**
 - OS: [macOS]
 - Browser [Chrome]

**Smartphone (please complete the following information):**
NA

**Additional context**
NA

**Comment by Link196** [comment]
I think I fixed it. I just renamed it from myNav to MyNav, and those warnings don't show up anymore. I think the reason we didn't spot this sooner was because the myNav constant wasn't being used for some reason.

**Comment by mgamlem3** [comment]
Fixed in #87
